react/features/filmstrip/components/web/Filmstrip.js

```diff
@@ -319,39 +319,47 @@ class Filmstrip extends PureComponent <Props, State> {
         const tileViewActive = _currentLayout === LAYOUTS.TILE_VIEW;
 
         switch (_currentLayout) {
+        case LAYOUTS.STAGE_FILMSTRIP_VIEW: {
+            if (_stageFilmstrip) {
+                if (_visible) {
+                    filmstripStyle.maxWidth = `calc(100% - ${_verticalViewMaxWidth}px)`;
+                }
+            } else {
+                filmstripStyle.maxWidth = _verticalViewMaxWidth;
```

Comment:
This code is exactly the same as the vertical filmstrip use case. Maybe if we convert the whole switch to `if`s we would be able to remove the duplications. WDYT?